cnxdb/archive-sql/get-books-containing-page.sql

```diff
@@ -0,0 +1,25 @@
+-- ###
+-- Copyright (c) 2013, Rice University
+-- This software is subject to the provisions of the GNU Affero General
+-- Public License version 3 (AGPLv3).
+-- See LICENCE.txt for details.
+-- ###
+
+-- arguments: page_uuid:string; page_version:string
+WITH RECURSIVE t(node, title, parent, path, value) AS (
+  SELECT nodeid, title, parent_id, ARRAY[nodeid], documentid
+  FROM trees tr, modules m
+  WHERE m.uuid = %(page_uuid)s::uuid
+  AND module_version(m.major_version, m.minor_version) = %(page_version)s
```

Comment:
Can you change these from the `page_` prefix to `document_` or no prefix at all. At this layer of the code most things are blissfully unaware of the page and book terminology.
It might seem trivial, but it's conversationally understood (hopefully) that pages and books are front-end terminology. Thus when in discussion one can switch to this terminology and generally be understood that the context has switched from back-end to front-end. That's at least the intention of the terminology separation.